boule/ellipsoid.py

```diff
@@ -443,6 +443,15 @@ def normal_gravity(self, latitude, height):
             The normal gravity in mGal.
 
         """
+        """
+        Warn if given point is not outside the ellipsoid
+        
+        """
+        if height < 0:
+            warn(f"The normal gravity formulas used are valid for points outside
```

Comment:
The errors are coming from breaking this string into multiple lines. See https://github.com/fatiando/boule/blob/0ecbb669596c67c3856384ad8b8148337fec23d3/boule/ellipsoid.py#L109 for an example of how to do it (this is one of the annoying things in Python 🙂 )